code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -182,4 +182,16 @@
 	name = "BBQ Sauce"
 	id = /datum/reagent/consumable/bbqsauce
 	results = list(/datum/reagent/consumable/bbqsauce = 5)
-	required_reagents = list(/datum/reagent/ash = 1, /datum/reagent/consumable/tomatojuice = 1, /datum/reagent/medicine/salglu_solution = 3, /datum/reagent/consumable/blackpepper = 1)
\ No newline at end of file
+	required_reagents = list(/datum/reagent/ash = 1, /datum/reagent/consumable/tomatojuice = 1, /datum/reagent/medicine/salglu_solution = 3, /datum/reagent/consumable/blackpepper = 1)
+	
+/datum/chemical_reaction/snow
+	name = "snow"
+	id = "snow"
+	required_temp = 273
+	required_reagents = list(/datum/reagent/water = 30)
```

Comment:
This will probably fuck up any chemical reaction that requires water and a temp lower than 273

Reply:
Maybe make ethanol an inhibitor for the water freeze reaction to give a way out of that problem